### PR TITLE
Update Distributed Tracing Page with Performance Views

### PR DESCRIPTION
**Pull request by MimiDumpling**
With the new Performance Views feature, Distributed Tracing docs need an update. 

**BEFORE**:
image

**AFTER:**
image

### CODE REVIEW

File: src/collections/_documentation/performance/distributed-tracing.md [hunk 1]
```diff
@@ -68,6 +68,50 @@ When you have multiple projects collecting transaction events, Sentry utilizes "
 
 If you enable Performance collection for a large portion of your traffic, you may exceed your organization's [Quotas and Rate Limits]({%- link _documentation/accounts/quotas/index.md -%}).
 
+## Viewing Transactions
+
+View transaction events by clicking on the "Transactions" pre-built query in [LINK: Discover], or by using a search condition `event.type:transaction` in a [LINK: Discover Query Builder] view.
```
Comment (markstory) [inline src/collections/_documentation/performance/distributed-tracing.md]:
```suggestion
You can view transaction events by clicking on the "Transactions" pre-built query in [LINK: Discover], or by using a search condition `event.type:transaction` in a [LINK: Discover Query Builder] view.
```
Might not be necessary 🤷‍♂

**Review by dashed — APPROVED** [review]
Looks good to me 👍

File: src/collections/_documentation/performance/distributed-tracing.md [hunk 2]
```diff
@@ -68,6 +68,50 @@ When you have multiple projects collecting transaction events, Sentry utilizes "
...
+
+The first top-level span is the transaction span, which encompasses all other spans within the transaction.
+
+**Zooming In on a Transaction**
+
+As displayed in the Discover span screenshot above, you can click and drag your mouse cursor across the minimap (top of the span view). You can also adjust the window selection by dragging the handlebars (black dashed lines). 
+
+**Missing Instrumentation**
+
+Sentry may indicate that gaps between spans are "Missing Instrumentation." This could mean that the SDK was unable to capture or instrument any spans within this gap automatically. It may require you to instrument the gap [manually](#setting-up-tracing).
```
Comment (PeloWriter) [inline src/collections/_documentation/performance/distributed-tracing.md:95]: Suggest "This message could mean ..."

File: src/collections/_documentation/performance/distributed-tracing.md [hunk 3]
```diff
@@ -68,6 +68,50 @@ When you have multiple projects collecting transaction events, Sentry utilizes "
...
+
+**Search by Trace ID**
+
+You can search using `trace id` by expanding any of the span details and click on "Search by Trace".
+
+You need **project permissions for each project** to see all the transactions within a trace. Each transaction in a trace is likely a part of different projects. If you don't have project permissions, some transactions may not display as part of a trace.
+
+**Traversing to Child Transactions**
+
+Child transactions are shown based on your project permissions -- which are necessary to viewing transaction events.
```
Comment (PeloWriter) [inline src/collections/_documentation/performance/distributed-tracing.md:111]: Add link to permission settings here?

**Pull request merged**